app/models/course/assessment/answer/scribing_scribble.rb

```diff
@@ -0,0 +1,5 @@
+# frozen_string_literal: true
+class Course::Assessment::Answer::ScribingScribble < ActiveRecord::Base
+  belongs_to :answer, class_name: Course::Assessment::Answer::Scribing.name, inverse_of: :scribbles
+  belongs_to :user, inverse_of: nil, foreign_key: true
```

Comment:
Hmm, the use case of the scribbles here is to see which user it belongs to. 

E.g. on a scribing answer, the student draws some scribbles and submits the answer. The teacher then adds his/her own set of scribbles on the answer (but he/she cannot edit the student's scribbles). If let's say the answer is unsubmitted by the teacher, then the student can see the teacher's set of newly added scribbles (but cannot edit). Meanwhile, the student can still edit the answer scribbles.

Like that still need the creator and updater columns?